src/MessageBox.Avalonia/ViewModels/Base/AbstractMsBoxViewModel.cs

```diff
@@ -18,17 +18,23 @@ public abstract class AbstractMsBoxViewModel:INotifyPropertyChanged
 
     {
     public bool CanResize { get; }
-    public bool HasHeader => !(string.IsNullOrEmpty(ContentHeader));
-    public bool HasIcon => !(ImagePath is null);
+    public bool HasHeader => !string.IsNullOrEmpty(ContentHeader);
+    public bool HasIcon => ImagePath is not null;
```

Comment:
Keyword `not` are required to use C# language standard 9.0 minimum, it means you have to use .NET 5 at least. I advice to use previous method instead of use keyword `not` for support .NET Core 2.1 and .NET Core 3.1 minimum.

Docs reference: https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/configure-language-version#defaults

Reply:
Sorry didn't notice you are on netstandard2.0. I will "try" to patch it (This is my first ever git PR ^^)